ledger/eval/eval.go

```diff
@@ -788,6 +791,39 @@ func StartEvaluator(l LedgerForEvaluator, hdr bookkeeping.BlockHeader, evalOpts
 		return nil, fmt.Errorf("overflowed subtracting rewards for block %v", hdr.Round)
 	}
 
+	// Move last block's proposer payout to the proposer
+	if !prevHeader.Proposer.IsZero() {
+		// Payout the previous proposer.
+		if !prevHeader.ProposerPayout.IsZero() {
+			// Use the FeeSink from that header, since that's the checked
+			// balance. (Though we have no expectation it will ever change.)
+			err = eval.state.Move(prevHeader.FeeSink, prevHeader.Proposer, prevHeader.ProposerPayout, nil, nil)
```

Comment:
as discussed offline, need to check either check proposer for existence or `ProposerPayout > minBalance`, or move this to `endOfBlock` (where no minBalance check will be performed).

Reply:
Payouts are no longer paid to accounts that have closed out.